src/core/api/payment_transactions_gateway_api/internal/v1/_openapi.json.tpl

```diff
@@ -1832,6 +1876,30 @@
           }
         }
       },
+      "VposDeleteResponse409": {
+        "type": "object",
+        "required": [
+          "requestId",
+          "status",
+          "error"
+        ],
+        "properties": {
+          "requestId": {
+            "type": "string",
+            "example": "77e1c83b-7bb0-437b-bc50-a7a58e5660ac"
+          },
+          "status": {
+            "type": "string",
+            "enum": [
+              "DENIED"
```

Comment:
If the only status for 409 response is `DENIED` maybe status field can be removed since it does not add any additional information on error code. What do you think?